README.md

```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
 We cover material in logical order, so that if you do read the earlier sections first,
 later sections on the interactions of tax and compensation will be clearer.
 
-The first sections are about how companies work and how they grow:
+We start with [**Equity Compensation Basics**](#equity-compensation-basics): What compensation
+and equity compensation are.
+But before we get much further, we need to talk about what stock is, and how companies are
+formed.
 
-- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): To understand
-  exactly what equity is, it’s essential to know how companies organize their ownership, how
-  stock works, and how ownership typically evolves.
-- [**Startups and Growth**](#startups-and-growth): While not everyone reading this works at an early
-  stage company, those who do can benefit from understanding the particulars of how these
-  companies raise money and grow.
+[**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations) covers how companies
+organize their ownership, how stock is issued, public and private companies, and IPOs and
+liquidity.
```

Comment:
Consider, "public and private companies, and how the fate of private companies affects equity." (Could put "IPOs and liquidity" as a parenthetical there. Just establish the relationship b/n each of these here to smooth out the logical narrative.)

Reply:
The reason for this is also the mouseovers. I'll adjust language.